kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -170,6 +180,22 @@
       "If true, no fields in any produced BigQuery schema will be REQUIRED. All "
       + "non-nullable avro fields will be translated as NULLABLE (or REPEATED, if arrays).";
 
+  public static final String INCLUDE_CONDITION_CONFIG = "includeCondition";
+  private static final ConfigDef.Type INCLUDE_CONDITION_TYPE = ConfigDef.Type.STRING;
+  private static final Boolean INCLUDE_CONDITION_DEFAULT = null;
+  private static final ConfigDef.Importance INCLUDE_CONDITION_IMPORTANCE =
+          ConfigDef.Importance.MEDIUM;
+  private static final String INCLUDE_CONDITION_DOC =
+          "If not null, only messages satifying the condition specified in the properties file will be passed on";
+
+  public static final String EXCLUDE_CONDITION_CONFIG = "excludeCondition";
+  private static final ConfigDef.Type EXCLUDE_CONDITION_TYPE = ConfigDef.Type.STRING;
+  private static final Boolean EXCLUDE_CONDITION_DEFAULT = null;
+  private static final ConfigDef.Importance EXCLUDE_CONDITION_IMPORTANCE =
+          ConfigDef.Importance.MEDIUM;
+  private static final String EXCLUDE_CONDITION_DOC =
+          "If not null, messages satifying the condition specified in the properties file will be dropped";
```

Comment:
it would be good to document whether include or exclude takes precedence to disambiguate cases where a condition may be true in both include and exclude case.